### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by yujinjcho**
For https://github.com/yujinjcho/rcruitme-front/issues/28

inspired a bit from https://www.bamboohr.com/ and http://www.greenhouse.io/

have some other sections to do, but can you take a look before I get too far ahead of myself

### CODE REVIEW

File: src/CreateAccount.js [hunk 1]
```diff
@@ -11,9 +10,9 @@ class CreateAccount extends Component {
     const baseUrl = "/sign-up"
     const url = redirect ? `${baseUrl}?redirect=${redirect}` : baseUrl;
     return (
-      <Nav.Item className="px-3">
-        <Link to={url}>Create Account</Link>
-      </Nav.Item>
+      <Nav.Link href={url} >
```
Comment (achiu8) [inline src/CreateAccount.js]: Why are we opting to do a full page refresh?

File: src/LandingPage.js [hunk 2]
```diff
@@ -1,11 +1,37 @@
-import React, { Component } from 'react';
+import React from 'react';
 import Container from 'react-bootstrap/lib/Container';
+import './mainSection.css';
```
Comment (achiu8) [inline src/LandingPage.js]: Let's rename this file to match the component. Typically try to have stylesheets map 1:1.

File: src/Navigation.js [hunk 3]
```diff
@@ -11,19 +10,23 @@ class Navigation extends Component {
...
     return (
-      <Navbar bg="light" expand="lg">
-        <Navbar.Brand>
-          <Link to='/' >Rcruitme</Link>
-        </Navbar.Brand>
-        <Nav className="mr-auto">
-        </Nav>
 
+      <Navbar collapseOnSelect expand="lg" bg="dark" variant="dark">
+        <Navbar.Brand href="/">
```
Comment (achiu8) [inline src/Navigation.js]: Similar question re: full reload vs. client-side navigation. Is this for styling reasons?

File: src/CreateAccount.js (hunk 1 above)
Comment (yujinjcho) [reply src/CreateAccount.js]: no particular reason

Comment (yujinjcho) [reply src/CreateAccount.js]: can update

**Pull request merged**